src/view/tables/I90Table.tsx

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import { CaseDetails } from "./CaseDetails";
+import { ChevronToggle } from "../util/ChevronToggle";
+import { connect } from "react-redux";
+import { Dispatch, AnyAction, bindActionCreators } from "redux";
+import { casesActionCreators, getCaseSummary } from "../../redux/modules/cases";
+import { RootState } from "../../redux/create";
+import { ELIS_CASE_BASE_URL, SNOOZE_OPTIONS } from "../../controller/config";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import ReactTooltip from "react-tooltip";
+import DateUtils from "../../utils/DateUtils";
+import NoteUtils from "../../utils/NoteUtils";
+import { SnoozeFormWrapper } from "../../controller/SnoozeFormWrapper";
+import { UpdateSnoozeFormWrapper } from "../../controller/UpdateSnoozeFormWrapper";
+import { appStatusActionCreators } from "../../redux/modules/appStatus";
+import "./ReceiptDisplayRow.scss";
+
+const mapStateToProps = (state: RootState) => ({
+  caselist: state.cases.caselist,
+  caseType: state.cases.type,
+  isLoading: state.cases.isLoading
+});
+
+const mapDispatchToProps = (dispatch: Dispatch<AnyAction>) =>
+  bindActionCreators(
+    {
+      toggleDetails: casesActionCreators.toggleDetails,
+      updateSummaryData: getCaseSummary,
+      setError: appStatusActionCreators.setDataLoadError,
+      setNotification: appStatusActionCreators.setNotification,
+      removeCase: casesActionCreators.removeCase,
+      onSnoozeUpdate: casesActionCreators.updateSnooze
+    },
+    dispatch
+  );
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+export const UnconnectedI90Table: React.FC<Props> = ({
+  caselist,
+  caseType,
+  isLoading,
+  toggleDetails,
+  updateSummaryData,
+  setError,
+  setNotification,
+  removeCase,
+  onSnoozeUpdate
+}) => {
+  if (caselist.length === 0 && isLoading) {
+    return null;
+  }
+
+  if (caselist.length === 0 && !isLoading) {
+    return <p>No cases found.</p>;
+  }
+
+  const numberOfColumns = 10;
+
+  const caseAge = (r: Case) => {
```

Comment:
seems overkill for this PR but I would like to have each one of these column renderers as its own component. I think it would allow use to easily expand to the next case and would allow us to document them on storybook